### PR TITLE
torus: add travis.yml

### PR DESCRIPTION
**Pull request by barakmich**

### CODE REVIEW

**Comment by barakmich** [comment]
Fixes #26

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+language: go
+
+sudo: false
+
+go:
+  - 1.5
+  - 1.6
+  - tip
+
```
Comment (xiang90) [inline .travis.yml]: add 
```
matrix:
  allow_failures:
    - go: tip
```

?

**Comment by barakmich** [comment]
Nah, it was go1.5 that failed (because no vendorexperiment by default).

I'm okay with current-go and tip.

**Comment by xiang90** [comment]
@barakmich Sometime go-tip breaks stuff. Then it might block merges. So in general, I would like just to ignore errors on go-tip.

**Comment by barakmich** [comment]
Yep. Makes sense. Hooray CI!

**Comment by klausenbusk** [comment]
Did you forget to close this issue?

**Pull request merged**